src/content_handlers/script_handler/src/script_handler.cpp

```diff
@@ -559,41 +558,44 @@ static ADUC_Result ScriptHandler_PerformAction(const std::string& action, const
     }
 
     exitCode = ADUC_LaunchChildProcess(adushconst::adu_shell, aduShellArgs, scriptOutput);
+
+    if (!scriptOutput.empty())
+    {
+        Log_Info(scriptOutput.c_str());
+    }
+
     if (exitCode != 0)
     {
         int extendedCode = ADUC_ERC_SCRIPT_HANDLER_CHILD_PROCESS_FAILURE_EXITCODE(exitCode);
         Log_Error(
-            "Child process failed (%s), extendedResultCode:0x%X (exitCode:%d)",
+            "Script failed (%s), extendedResultCode:0x%X (exitCode:%d)",
             action.c_str(),
             extendedCode,
             exitCode);
         result = { .ResultCode = ADUC_Result_Failure, .ExtendedResultCode = extendedCode };
```

Comment:
goto done after this line so don't need else on line 577 below either.
Also, should it do workflow_set_result_details like on line 585 too, or no?

Reply:
Fix. No need for result details here.